src/main/java/com/github/dockerjava/core/command/CreateNetworkCmdImpl.java

```diff
@@ -58,6 +58,9 @@ public CreateNetworkCmd withDriver(String driver) {
 
     @Override
     public CreateNetworkCmd withIpamConfig(Ipam.Config config) {
+        if (this.ipam == null) {
+            this.ipam = new Ipam();
+        }
```

Comment:
It's an optional config to create a network
https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#create-a-network
If I don't set it then the next line will raise NPE, for me it's a bug that prevent user to set custom configuration for network creation.